### PR TITLE
Merge IntegrationTests and RNTester with Stock React Native 0.60.6

### PR DESCRIPTION
**Pull request by NickGerleman**
This gets us a little bit closer to stock React Native, removing MacOS and Android specific files that break flow-check when using unforked RN. This change started as replacing everything, then going through the list of files with diffs to determine which diffs can be trivially reverted (are macOS or Android specific).

There are a few files with changes that we notably keep as is. These include:

- `AsyncStorageTest.js` which has been patched to not test merge functionality which we do not implement
- `IntegrationTestsApp.js` which currently comments out certain tests to avoid running them
- `FlatListExample.js` which has added functionality.
- `ListExampleShared.js` which has added functionality.
- `RNTesterExampleContainer.js` which has some changes in platform logic.
- `RNTesterExampleList.js` which has added functionality
- `ScrollViewExample.js` which has a focus related change
- `TouchableExample.js` which has added functionality
- `ViewExample.js` which has added functionality
- `RNTesterTypes.js` which has a platform logic related change

As a followup I plan to move modified files out to be overrides, so we can keep these directories clean. This change was not manually validated.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4006)

### CODE REVIEW

**Comment by NickGerleman** [comment]
I think I can also remove the platform array related diffs when moving to overrides, since the change seems to be to allow supporting both iOS and macOS.

**Comment by NickGerleman** [comment]
Going to do the rest of the moving in the 61 topic branch to avoid extra work.

File: vnext/RNTesterCopy/js/FocusEventsExample.js [hunk 1]
```diff
@@ -1,217 +0,0 @@
-/**
```
Comment (NickGerleman) [inline vnext/RNTesterCopy/js/FocusEventsExample.js]: We don't reference this file in our RNTesterList or anywhere in our project, so guessing it was Mac only.

**Comment by ddalp** [comment]
Make sense since we are going to do manual copy/merge again for 61 anyway.

---
In reply to: [580607442](https://github.com/microsoft/react-native-windows/pull/4006#issuecomment-580607442) [](ancestors = 580607442)

**Pull request merged**